templates/search/includes/filter.html

```diff
@@ -13,7 +13,7 @@
                     <a href="{% url 'search-website-long-filter-chooser' field_name=field.name %}?{{ request.GET.urlencode }}"  aria-label='See more'>See more</a>
                 {% endif %}
             {% endif %}
-            <input type="submit" value="Update" class="search-filters__submit">
+<!--            <input type="submit" value="Update" class="search-filters__submit">-->
```

Comment:
This comment is starting from 1st col, it could be aligned under closing div to facilitate collapsing of html attributes in the editor.